### ISSUE TITLE
Not display translation language

### ISSUE THREAD

**Issue by luatdolphin** [bbepis/XUnity.AutoTranslator#172]
This game has a PC version. You can download from this https://ty.qq.com/
I tried using this tool for this game, but It cannot translate this game. It just translates in first windows ( the first window is update windows, but it doesn't relate to the core of the game). After updating, The main window of the game will be launched, I tried to patch it, but nothing to respond.
![image](https://user-images.githubusercontent.com/2277080/115139525-74168380-a05c-11eb-932e-2e3669fe7ab8.png)
![image](https://user-images.githubusercontent.com/2277080/115139533-7f69af00-a05c-11eb-9600-7750ded444aa.png)
Can you help me with this situation? I really appreciate it!

**Comment by gravydevsupreme** [comment]
All I see in the screenshot is a whole lot of DLLs that are not used by the Unity Engine. Is this one of those games that download the binaries on startup like for example Genshin Impact? Does it have a managed folder with DLLs?

I have not been able to download the game because I can't really navigate the site properly. If it does work like Genshin Impact though, I am not sure there is plugin loader that works with it.

**Comment by luatdolphin** [comment]
Yes. It has a managed folder with DLLs. I think it does work like Genshin Impact, but I'm not sure
![image](https://user-images.githubusercontent.com/2277080/115321282-b3081e80-a1ad-11eb-86e6-5d5d7c619ab4.png)
This is the download link of game that I uploaded to mirror
https://gofile.io/d/4xfmG7

**Comment by gravydevsupreme** [comment]
Took a look at the game and had a discussion with a BepInEx developer:
 1. The initial windows of the games that can be translated are mono-based.
 2. The main-game is IL2CPP-based.

Therefore it would require a different plugin loading setup for the different parts. But the current version of BepInEx (even BE builds of IL2CPP) cannot hook properly into it.

So for now, it will not work for this game.

**Comment by luatdolphin** [comment]
Thanks a lot!
